Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ThemeListener/ThemeListenerPage.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<Page
```

Comment:
Remove this sample. This should be added once the sample app supports dark theme